src/server/sql/user/create_users_table.sql

```diff
@@ -4,7 +4,8 @@
 
 CREATE TABLE IF NOT EXISTS users(
   id SERIAL PRIMARY KEY NOT NULL,
-  email VARCHAR(254) UNIQUE,
+  username VARCHAR(254) UNIQUE,
```

Comment:
This change needs a migration. Let me know if you need help with this.

Reply:
Ok I added the changes, I beleive just renaming the email column to username should be all that is needed.  Let me know if I missed something.